src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/WhyCommand/IWhyCommandRunner.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal interface IWhyCommandRunner
+    {
+        Task ExecuteCommand(WhyCommandArgs whyCommandArgs);
```

Comment:
I realized that I might have been using a `Task` return type unnecessarily here. I don't believe this method needs to be async at all, but I could be wrong. I've now changed it to return an `int` directly, instead of a `Task`. Let me know if I've got this wrong